pkgs/development/libraries/libspnav/configure-socket-path.patch

```diff
@@ -0,0 +1,39 @@
+diff --git a/spnav.c b/spnav.c
+index f9e10f8..f9a117b 100644
+--- a/spnav.c
++++ b/spnav.c
+@@ -36,7 +36,8 @@ OF SUCH DAMAGE.
+ #include <sys/select.h>
+ #include "spnav.h"
+ 
+-#define SPNAV_SOCK_PATH "/var/run/spnav.sock"
++#define DEFAULT_SPNAV_SOCK_PATH "/var/run/spnav.sock"
```

Comment:
`/var/run` is a symlink to `/run`. Using `/var/run` is discouraged. If we touch this line anyways, we should change it to there.